idioms/return-moved-arg-on-error.md

```diff
@@ -0,0 +1,60 @@
+# A succinct name for the pattern
+
+## Description
+
+If fallible method receives consumes argument, return it back inside error.
```

Comment:
You need to drop either `receives` or `consumes`. And maybe add a few `a`s and `an`s. Also, we're usually talking about `function`s rather than `methods`. Suggestion:

```suggestion
If a fallible function consumes an argument, return that argument back inside an error.
```

Another thing which you may want to highlight is that you usually do this for functions mutating or converting some operand.